lib/ansible/modules/network/nxos/nxos_nxapi.py

```diff
@@ -94,6 +94,39 @@
     choices: ['present', 'absent']
     required: false
     default: present
+  ssl_strong_ciphers:
+    description:
+      - Controls the use of whether strong or weak ciphers are configured.
+        By default, this feature is disabled and weak ciphers are
+        configured.  To enable the use of strong ciphers, set the value of
+        this argument to True.
+    required: false
+    default: no
```

Comment:
Here and for the other new `bool` parameters, I would update the default value to be `False`/`True` instead of `no`/`yes` so that it matches what is set in the argument spec in `main()`